chia/plotting/manager.py

```diff
@@ -232,9 +232,9 @@ def _refresh_task(self, sleep_interval_ms: int):
 
                 plot_filenames: Dict[Path, List[Path]] = get_plot_filenames(self.root_path)
                 plot_directories: Set[Path] = set(plot_filenames.keys())
```

Comment:
The `.keys` call is also redundant here, as
```python
set({'foo': [1, 2], 'bar': [2]}) == {'foo', 'bar'}
```

Reply:
Doesn't hurt though, makes it at least obvious.

Reply:
> It would actually suffice to just pass `set(plot_filenames)` to the one function which consumes it, 45 lines below.

Hm then you create the set in the loop for each batch though, im not sure about the cost of that but the end that line you commented on is also unrelated to this PR, isn't it? :)